sentry-types/src/protocol/envelope.rs

```diff
@@ -1,14 +1,21 @@
-use std::{io::Write, path::Path};
+use std::{
+    io::{BufRead, Write},
+    path::Path,
+    time::SystemTime,
+};
 
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
 use uuid::Uuid;
 
-use super::v7 as protocol;
+use crate::utils::ts_rfc3339_opt;
+use crate::Dsn;
+
+use super::v7::{self as protocol};
```

Comment:
Not sure why this change is needed (surprising though if this wasn't caught by rustfmt or clippy)

```suggestion
use super::v7 as protocol;
```